.github/workflows/antithesis.yml

```
@@ -0,0 +1,118 @@
+      contents: read
+      attestations: write
+      id-token: write
+
+    steps:
+    - name: 🚧 Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: 📥 Checkout repository
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.inputs.ref_name || '' }}
```

actionlint (1.7.8)

property "ref_name" is not defined in object type {duration: string; no-faults: string; test: string}

(expression)